File: Periode 3/Java/Chapter 1/1. Classes/Exercise5/src/Interval.java
/**
 * Created by Joost on 23-2-2017.
 */
public class Interval
{
    public int Begin;
    public int Eind;

    public Interval(int begin, int eind)
    {
        this.Begin = begin;
        this.Eind = eind;
    }

    public int Sum()
    {
        int output = 0;
        for(int i = Begin; i <= Eind; i++)
        {
            output += i;
        }
        return output;
    }

    public String Product()
    {
        String output = "";
        for(int i = (Begin+1); i < Eind; i++)
        {
            output += i;
        }
        return output;
    }
}

File: Periode 3/Chapter 1 - Done/1 Classes - Done/Exercise5/Interval.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Exercise5
{
    class Interval
    {
        public int Begin;
        public int End;

        public Interval(int Begin, int End)
        {
            this.Begin = Begin;
            this.End = End;
        }

        public int Sum()
        {
            int output = 0;
            for (int i = this.Begin; i <= this.End; i++)
            {
                output = output + i;
            }
            return output;
        }

        public string Product()
        {
            string output = "";
            for (int i = (this.Begin+1); i < this.End; i++)
            {
                output = output + i;
            }
            return output;
        }
    }
}
